extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -71,6 +72,12 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 			\Wikia::addAssetsToOutput( 'community_page_benefits_js' );
 			\Wikia::addAssetsToOutput( 'community_page_benefits_scss' );
 		}
+
+		if ( !$out->getUser()->isAnon() ) {
```

Comment:
@lukaszkonieczny I can't check if the cookie is set here, because it will break the functionality for VE. With VE the workflow goes like this:

* Inject assets on page load
* On PHP side, set cookie in onUserFirstEditOnLocalWiki
* mw.hook('postEdit') is fired, check for cookie there and unset if necessary.